metrics/sinks/influxdb/influxdb_test.go

```diff
@@ -37,9 +37,10 @@ type fakeInfluxDBDataSink struct {
 
 func newRawInfluxSink() *influxdbSink {
 	return &influxdbSink{
-		client:  influxdb_common.Client,
-		c:       influxdb_common.Config,
-		conChan: make(chan struct{}, influxdb_common.Config.Concurrency),
+		client:   influxdb_common.Client,
+		c:        influxdb_common.Config,
+		dbExists: true,
```

Comment:
The value for `dbExists` should be `false` since there is a check at the beginning of the function. https://github.com/kubernetes/heapster/pull/2086/files#diff-a83c1db1f245f8fb4c41e99889717525R257

Reply:
if set dbExists to true, then the unittest will try to create a database and failed.

Reply:
Then you need to fix the ut. Make sure you implement the logic correct first and adjust your ut.